examples/livesync/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:18
+
+ENV IS_DOCKER=true
+WORKDIR /app
+RUN curl -f https://get.pnpm.io/v6.16.js | node - add --global pnpm
+COPY . .
+RUN pnpm install
+COPY ./docker-entrypoint.sh /
+RUN chmod +x /docker-entrypoint.sh
+ENTRYPOINT ["/docker-entrypoint.sh"]
```

Comment:
Why do we need a separate `ENTRYPOINT` and `CMD`? Seems to me unclear how the user is intended to start the container.

Instead we could just do a `RUN pnpm build` and a `CMD pnpm seed && pnpm start` ?

But - do we always want to seed the database when starting the app container? Istm we should do that when we start the postgres container instead?